bagua/torch_api/communication.py

```diff
@@ -38,46 +38,61 @@ def is_initialized():
     return _global_state is not None
 
 
-def init_process_group(init_method: str = "dist://", device_id=None):
+def init_process_group():
+    """Initializes the default distributed process group, and this will also
+    initialize the distributed package, should be executed before all the APIs
```

Comment:
the PyTorch builtin distributed process group?

Reply:
DONE